src/OrchardCore.Modules/OrchardCore.ContentLocalization/Views/LocalizationPart.SummaryAdminLinks.cshtml

```diff
@@ -7,16 +7,19 @@
             <span>@T["Localizations"]</span>
         </button>
         <ul class="dropdown-menu dropdown-menu-end">
+            @{
+                var returnUrl = $"{Context.Request.Path}{Context.Request.QueryString}";
```

Comment:
Please move it to the top of the file, also no need to string interpolation

Reply:
done